hugo.yaml

```diff
@@ -23,6 +23,10 @@ languages:
     languageName: 中文 (Chinese)
     params:
       description: OpenTelemetry 项目网站
+  pt-br:
+    languageName: Português
```

Comment:
```suggestion
    languageName: Português do Brasil
```

Reply:
Não precisamos explicitar que é `Português do Brasil`.
Olhando alguns exemplos de documentação multilinguagem AWS, K8S apenas a apalavra `português` é exibida no menu da página. Apesar das pequenas diferenças da gramática entre português de Portugal e brasileiro, acredito que isso não será um problema.

Reply:
Então @EzzioMoreira, eu voto por começar já explicitando que é português do Brazil, tivemos alguns problemas nas docs do k8s relacionados a isso (até está nascendo um time pt-pt lá), e nas configurações do Hugo multilingual tem algumas diferenças consideráveis entre pt-pt e pt-br.

Nesse caso em específico que é só o título da aba de idioma, eu diria pra seguir o padrão dos outros idiomas (que tem a escrita no idioma em original e outra em inglês) ou fazer algo assim: "Português (Brazil)", o que acham?

Reply:
@edsoncelio, thanks for sharing your experiences from K8s.

Let me ask this question: how likely is it that we'll end up with two localization of the docs (`pt-pt` _and_ `pt-br`)? If it is not likely, I'd vote to use the simple language name "Português". The most important point is that the language key in the config file is `pt-br` (which it is currently).

Reply:
That being said, I'm ok with "Português (Brazil)" if that's what you all end up choosing.